infrastructure/xai-relayer-api/src/app.ts

```diff
@@ -0,0 +1,53 @@
+import express from "express";
+import { disconnectMongoose, loadMongoose } from "@/loaders/mongoose";
+import { loadExpress } from "@/loaders/express";
+import { resolve } from "path";
+import dotenv from "dotenv";
+dotenv.config({ path: resolve(__dirname, "../.env") });
+
+export const app = express();
+
+(async () => {
+	await loadMongoose();
+	await loadExpress();
```

Comment:
this isn't the best for scaling, it's fine if this api is only used a somewhat small amount of requests, but we've learned you want to establish and disconnect these as needed.

Reply:
I added a comment to this ticket
https://www.pivotaltracker.com/story/show/187699973

We will make it so all db services will open the connection if its not open on that machine so we don't start the connection with loading the app.